stwcs/wcsutil/altwcs.py

```diff
@@ -553,26 +541,7 @@ def convertAltWCS(fobj, ext, oldkey='', newkey=''):
     hdr: `astropy.io.fits.Header`
         header object with keywords renamed from oldkey to newkey
     """
-    if oldkey.strip() == '':
-        oldkey = ' '
-
-    hdr = readAltWCS(fobj, ext, wcskey=oldkey)
-    if hdr is None:
-        return None
-
-    # Converting WCS to new key
-    for kwd in hdr:
-        if oldkey == ' ':
-            kwd_new = kwd + newkey.strip()
-        elif kwd.endswith(oldkey):
-            kwd_new = kwd[:-1] + newkey.strip()
-        else:
-            # it is a keyword not specific to an alternative WCS. skip it:
-            continue
-
-        hdr.rename_keyword(kwd, kwd_new, force=True)
-
-    return hdr
+    return _read_alt_wcs(fobj, ext, wcskey=oldkey, outkey=newkey)
```

Comment:
I was considering this but this function may not convert anything if `outkey==wcskey` or if `outkey is None`. What about `_read_wcs_as_alt`, `_load_wcs_as_alt` (though it might be a primary if `outkey=' '` but then maybe primary is just a subset of "extended alt" (`'A'-'Z' AND ' '`)???)